sources/packages/api/src/services/sfas/sfas-part-time-application.service.ts

```diff
@@ -46,6 +46,51 @@ export class SFASPartTimeApplicationsService
     await this.repo.save(application, { reload: false, transaction: false });
   }
 
+  /**
+   * Validates before an application submission to see if there is an overlapping SFAS part-time application existing.
+   * @param sin
+   * @param birthDate
+   * @param lastName
+   * @param studyStartDate
+   * @param studyEndDate
+   * @returns SFAS part-time application.
+   */
+  async validateDateOverlap(
+    sin: string,
+    birthDate: Date,
+    lastName: string,
+    studyStartDate: Date,
+    studyEndDate: Date,
+  ): Promise<SFASPartTimeApplications> {
+    return this.repo
+      .createQueryBuilder("sfasPTApplication")
+      .select(["sfasPTApplication.id"])
+      .innerJoin("sfasPTApplication.individual", "sfasPTstudent")
+      .where("sfasPTstudent.lastName = :lastName", { lastName })
```

Comment:
Please do the comparison in a not case sensitive way. If I am not wrong the user last name already has a index for lower(lastName).

Reply:
lower function added.